x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/inference_pipeline_card.tsx

```diff
@@ -39,20 +37,18 @@ import { TrainedModelHealth } from './ml_model_health';
 import { MLModelTypeBadge } from './ml_model_type_badge';
 import { PipelinesLogic } from './pipelines_logic';
 
-export const InferencePipelineCard: React.FC<InferencePipeline> = (pipeline) => {
+export const TrainedModelHealthPopover: React.FC<InferencePipeline> = (pipeline) => {
   const { http } = useValues(HttpLogic);
   const { indexName } = useValues(IndexNameLogic);
   const { ingestionMethod } = useValues(IndexViewLogic);
+
+  const { deleteMlPipeline, detachMlPipeline } = useActions(PipelinesLogic);
+
   const [isPopOverOpen, setIsPopOverOpen] = useState(false);
   const [showConfirmDelete, setShowConfirmDelete] = useState(false);
-  const { deleteMlPipeline, detachMlPipeline } = useActions(PipelinesLogic);
-  const showConfirmDeleteModal = () => {
-    setShowConfirmDelete(true);
-    setIsPopOverOpen(false);
-  };
-  const { modelId, pipelineName, types: modelTypes } = pipeline;
-  const modelType = getMLType(modelTypes);
-  const modelTitle = getModelDisplayTitle(modelType);
+
+  const { pipelineName } = pipeline;
+
   const actionButton = (
     <EuiButtonEmpty
```

Comment:
I see some discrepancies with the model-related subcomponents here vs in the model selection component within the pipeline configuration flyout. Let's make sure we address these in the unified ML design:
* The status badge is part of the "..." button here. In the model selection list it's not.
* We display a wrench button to navigate to the Trained Models page if the model is not deployed vs in the model selection list we don't.
* The model status badge here works with the reduced set of states (implemented in `TrainedModelHealth`), for example the downloading/downloaded states all map to "Not started". If we want to display more fine-grained states, we'll need to use model details from `MlModel` in the pipeline card.
* The model type is rendered as a badge here (`text_embedding`) vs a user-friendly title in the model selector ("Dense Vector Text Embedding"). (We may not have the space the display the full title here.)

I'll link to this comment in the unified UX Jira.

Reply:
Ok, since this an open question for the UX folks, I will keep this as-is for now and we can adjust in a follow-up PR.